lib/chef/resource/windows_certificate.rb

```diff
@@ -230,13 +264,15 @@ def export_cert(cert_obj, cert_path)
           out_file.close
         end
 
-        def cert_location
```

Comment:
Minor, but `cert_location[0]` scans better than `get_cert_location[0]`.

Reply:
Lamont suggested the same thing. I added a second method and changed them to ps_cert_location and native_cert_location to reflect PowerShell versus the actual name of the store.